lib/mrsk/commands/builder/native.rb

```diff
@@ -1,17 +1,32 @@
 class Mrsk::Commands::Builder::Native < Mrsk::Commands::Builder::Base
   def create
-    # No-op on native
+    # No-op on native without cache
+
+    if config.builder.cache?
+      docker :buildx, :create, "--use", "--driver=docker-container"
+    end
   end
 
   def remove
-    # No-op on native
+    # No-op on native without cache
+
+    if config.builder.cache?
+      docker :buildx, :rm, builder_name
+    end
   end
 
   def push
-    combine \
-      docker(:build, *build_options, build_context),
-      docker(:push, config.absolute_image),
-      docker(:push, config.latest_image)
+    if config.builder.cache?
+      docker :buildx, :build,
+        "--push",
+        *build_options,
+        build_context
+    else
+      combine \
+        docker(:build, *build_options, build_context),
+        docker(:push, config.absolute_image),
+        docker(:push, config.latest_image)
+    end
   end
```

Comment:
Instead of doing conditionals on all of these, I think maybe we should have `Mrsk::Commands::Builder::Native::Cached`?

Reply:
Good point!

Reply:
Done, added `Mrsk::Commands::Builder::Native::Cached` builder.